lib/ReactViews/Workbench/Controls/ChartItemSelector.jsx

```diff
@@ -53,11 +55,13 @@ const ChartItemSelector = observer(function({ item }) {
   // discretelytimevarying items and have a separate chart button to enable/disable.
   const chartItems = chartView.chartItems
     .filter(c => c.item === item)
-    .filter(c => c.type !== "momentPoints" && c.type !== "momentLines");
+    .filter(c => c.type !== "momentPoints" && c.type !== "momentLines")
+    .sort((a, b) => (a.name >= b.name ? 1 : -1));
+
```

Comment:
Sort alphabetically by name. This ensures a stable order for chart item selectors not affected by the order of items in array trait that changes when some trait is set.